app/(afterLogin)/(okrplanning)/okr/settings/target-assignment/_components/assign-target-drawer/index.tsx

```
@@ -53,14 +53,14 @@ const AssignTargetDrawer: React.FC = () => {
     });
   };
   useEffect(() => {
-    if (currentId && getTargetById) {
-      form.setFieldsValue({
-        department: getTargetById.departmentId,
-        criteria: getTargetById.vpCriteriaId,
-        month: [getTargetById.month],
-        [getTargetById.month]: getTargetById.target,
-      });
-      setSelectedMonths([getTargetById.month]);
+          if (currentId && getTargetById) {
```

ESLint

[error] Delete `······`

(prettier/prettier)